packages/chrome-extension/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+export function convertPublishableKeyToFrontendAPIOrigin(key = '') {
+  return `https://${atob(key.replace(/pk_(test|live)_/, '')).slice(0, -1)}`;
+}
+
+export async function getClientCookie(url: string) {
+  return chrome.cookies.get({ url, name: '__client' });
```

Comment:
Can these throw? If so, should we be handling this or not?

Reply:
The docs does not mention anything about throwing errors. One case i could think of is if someone uses `@clerk/chrome-extension` and forgets to add the `cookies` and `storage` permissions in their `manifest.json` (probably would cause permission errors to be raised).
I would suggest we add a error handler for those 2 cases and make the action log the error without causing the extension to crash.
see: https://github.com/clerkinc/javascript/pull/971/commits/11df68a6a9d663774fac8ef0143559d39d66e9d9
WDYT?
cc: @SokratisVidros

Reply:
Im fine with this approach for now :)